go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/coreos/go-systemd v0.0.0-20190620071333-e64a0ec8b42a // indirect
 	github.com/emicklei/go-restful v2.9.6+incompatible // indirect
 	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
+	github.com/go-logfmt/logfmt v0.4.0 // indirect
```

Comment:
Could you try to run `go mod tidy`? No idea where this comes from, because you didn't change any dependencies.

Everything else lgtm.

Reply:
done, thanks